cli/tests/integration/repl_tests.rs

```diff
@@ -131,24 +131,43 @@ fn pty_complete_expression() {
 
 #[test]
 fn pty_complete_imports() {
-  util::with_pty(&["repl", "-A"], |mut console| {
```

Comment:
I updated the pty tests to use a temporary cwd by defualt in `util::with_pty` because otherwise now it will load too many files in that may conflict with things such as completions. Actually, thinking about it more, I should probably do the same for the lsp tests.

Reply:
I agree, it makes more sense to use temp CWD here and in the LSP tests